src/components/Card.tsx

```diff
@@ -88,15 +92,18 @@ const cardContainer = (
       'py-5': paddingType === 'normal',
       'py-8': typeof paddingType === 'undefined',
     }),
-    width(
-      thin ? 'sm:!w-thin-card' : 'sm:w-card',
-      thin ? 'tiny:w-thin-mobile' : 'w-mobile-card',
-      { 'w-32': thin }
-    ),
+    !useAppStyles
```

Comment:
I think that it's hard to read maybe you could split this into constsants?

Reply:
good idea, tyty